deploy/clowdapp.yaml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ingress
+objects:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: marketplace-tmp-claim
+    labels:
+      app: marketplace
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: "${VOLUME_CAPACITY}"
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ingress
```

Comment:
Probably want to change the app name (as well as the template name)

Reply:
yep, missed this one.